README.md

```diff
@@ -27,10 +27,10 @@
 <p align="center">
   <a href="#chatops">ChatOps</a> •
   <a href="#why-use-opsdroid">Why use opsdroid?</a> •
-  <a href="#quickstart">Quick Start</a> •
+  <a href="#quick-start">Quick Start</a> •
   <a href="#installation-guide">Installation Guide</a> •
   <a href="#usage">Usage</a> •
-  <a href="#contributors">Contributors</a> •
+  <a href="#contributing">Contributors</a> •
```

Comment:
hash part generated via markdown and will end up by the following user nowhere, more of that the actuall contributing docs located just under `contributors` section.